platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/result/DefaultResolvedArtifactResult.java

```diff
@@ -65,7 +81,17 @@ public File getFile() {
     }
 
     @Override
+    @Deprecated
```

Comment:
If we're going to deprecate, we should also nag. I have a strong feeling that this method is used in a lot of places, including AGP. I think we should be cautious about changing this API